custom/ewsghana/api.py

```diff
@@ -702,7 +706,7 @@ def web_user_sync(self, ews_webuser):
         return user
 
     def _reassign_number(self, user, connection):
-        v = VerifiedNumber.by_phone(connection.phone_number, include_pending=True)
+        v = VerifiedNumber.by_phone(apply_leniency(connection.phone_number), include_pending=True)
```

Comment:
```by_phone``` method only strip plus from phone number and I've just found number with whitespaces in v1 so it's safer to apply leniency.